src/test/scala/mesosphere/marathon/integration/LeaderIntegrationTest.scala

```diff
@@ -17,7 +29,8 @@ class LeaderIntegrationTest extends IntegrationFunSuite with MarathonClusterInte
 
   test("the leader abdicates when it receives a DELETE") {
     Given("a leader")
-    val leader = marathon.leader().value
+    eventually { marathon.leader.code should be (200) }
```

Comment:
The response will not change, after it is created. eventually trying again does not make sense.